src/pages/index.tsx

```diff
@@ -0,0 +1,415 @@
+                </p>
+                <p>The following example creates a http server.</p>
+                <p>We pass the relevant typings for our http version used.</p>
+                <p>By passing types we get correctly typed access to the underlying http objects in routes.</p>
+                <p>
+                  If using http2 we&39;d pass
+                  <CustomHighLight text={'<http2.Http2Server, http2.Http2ServerRequest, http2.Http2ServerResponse>'} />.
+                </p>
+                <p>
+                  For https pass <CustomHighLight text={'http2.Http2SecureServer'} />
+                  or <CustomHighLight text={'http.SecureServer'} /> instead of Server.
+                </p>
+                <p>
+                  This ensures within the server handler we also get <CustomHighLight text={'http.ServerResponse'} />
+                  with correct typings on <CustomHighLight text={'reply.res'} />.
+                </p>
+                <div className="flex justify-end">
+                  <span>async/await</span> <ToggleSwitch />
+                </div>
+                {isToggled ? <TypeScriptSupportWithAsyncAwait /> : <TypeScriptSupport />}
+
+                <p>
+                  Visit the
+                  <a href="https://www.fastify.io/docs/latest" aria-label="Documentation">
+                    Documentation
+                  </a>
+                  to learn more about all the features that Fastify has to offer.
+                </p>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="section alternate">
+          <div className="container content">
+            <div className={styles.responsiveContainer}>
+              <div>
+                <h1 className="title">A fast web framework</h1>
+                <p>
+                  Leveraging our experience with Node.js performance, Fastify has been built from the ground up to be
+                  <strong>&nbsp;as fast as possible</strong>. Have a look at our
+                  <a href="/benchmarks">&nbsp;benchmarks section</a> to compare fastify performance to other common web
+                  frameworks.
+                </p>
+                <Button label={'Check out our benchmarks'} />
+              </div>
+              <div className="spacer"></div>
+              <div>
+                <h1 className="title">Ecosystem</h1>
+                <p>
+                  Fastify has an ever-growing ecosystem of plugins. Probably there is already a plugin for your
+                  favourite database or template language. Have a look at the <a href="/ecosystem">Ecosystem page</a> to
+                  navigate through the currently available plugins. Can&39;t you find the plugin you are looking for? No
+                  problem, <a href="/docs/master/Plugins">it&39;s very easy to write one</a>!
+                </p>
+                <Button label={'Explore 253 plugins'} />
```

Comment:
It misses the `onclick` function

Anyway we can use the standard docusaurus style class
https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima

https://infima.dev/docs/components/button

Reply:
alright...let me read through...